test/utils/image/manifest.go

```diff
@@ -42,6 +42,7 @@ type RegistryList struct {
 	PrivateRegistry         string `yaml:"privateRegistry"`
 	SampleRegistry          string `yaml:"sampleRegistry"`
 	MicrosoftRegistry       string `yaml:"microsoftRegistry"`
+	DockerLibraryRegistry   string `yaml:"dockerLibraryRegistry"`
```

Comment:
As per the issue, you may need to override this value so I don't see why it isnt customizable.